aspnetcore/security/authentication/azure-ad-b2c-webapi.md

```diff
@@ -0,0 +1,208 @@
+* [Visual Studio 2017](https://aka.ms/vsdownload?utm_source=mscom&utm_campaign=msdocs) (any edition)
+* [Postman](https://www.getpostman.com/postman)
+
+## Create the Azure Active Directory B2C tenant
+
+Create an Azure Active Directory B2C tenant [as described in the documentation](/azure/active-directory-b2c/active-directory-b2c-get-started). When prompted, associating the tenant with an Azure subscription is optional for this tutorial.
+
+## Configure a sign-up or sign-in policy
+
+Use the steps in the Azure AD B2C documentation to [create a sign-up or sign-in policy](/azure/active-directory-b2c/active-directory-b2c-reference-policies#create-a-sign-up-or-sign-in-policy). Name the policy **SiUpIn**.  Use the example values provided in the documentation for **Identity providers**, **Sign-up attributes**, and **Application claims**. Using the **Run now** button to test the policy as described in the documentation is optional.
+
+## Register the API in Azure AD B2C
+
+In the newly created Azure AD B2C tenant, register your API using [the steps in the documentation](/azure/active-directory-b2c/active-directory-b2c-app-registration#register-a-web-api) under the **Register a web API** section. Stop at the **Create a web app client secret** section. A client secret isn't required for this part of tutorial. 
+
+Use the following values:
+
+| Setting                       | Value               | Notes                                                                                  |
+|-------------------------------|---------------------|----------------------------------------------------------------------------------------|
+| **Name**                      | *&lt;API name&gt;*  | Enter a **Name** for the app that describes your app to consumers.                     |
+| **Include web app / web API** | Yes                 |                                                                                        |
+| **Allow implicit flow**       | Yes                 |                                                                                        |
+| **Reply URL**                 | `https://localhost` | Reply URLs are endpoints where Azure AD B2C returns any tokens that your app requests. |
+| **App ID URI**                | *api*               | The URI doesn't need to resolve to a physical address. It only needs to be unique.     |
+| **Include native client**     | No                  |                                                                                        |
+
+After the API is registered, the list of apps and APIs in the tenant is displayed. Select the API that was just registered. Select the **Copy** icon to the right of the **Application ID** field to copy the Application ID to the clipboard.  Select **Published scopes** and verify the default *user_impersonation* scope is present.
```

Comment:
There's an extra space before the 4th sentence.

Reply:
Whoa, another unexpected comment from Scott! :D :D :D

Reply:
@rachelappel Just me proving my worth around here. 😅